### ISSUE TITLE
Jupyterlab 4.0.0 makes interactive Panel widgets not work 

### ISSUE THREAD

**Issue by aleave** [jupyterlab/jupyterlab#14581]
## Description

Interactive widgets built with the Panel library are no longer working properly in version 4.0.0: a change in the widget does not seem to trigger a recalculation of the cell. For example, having a selection widget linked to a chart and changing the selected value in the widget does not refresh the chart. See example below

## Reproduce

Open a new noteook, paste the code below in a cell, run it and try to change the value in one of the select widgets. The chart will not refresh as it normally should

```import pandas as pd
import panel as pn
import plotly.express as px

pn.extension('plotly')

# Let's assume this is your dataframe
df = pd.DataFrame({
    'A': ['foo', 'bar', 'foo', 'bar', 'foo', 'bar', 'foo', 'foo'],
    'B': ['one', 'one', 'two', 'three', 'two', 'two', 'one', 'three'],
    'C': [0, 1, 2, 3, 4, 5, 6, 7],
})

# Creating dropdowns
dropdown_A = pn.widgets.Select(name='A', options=list(df['A'].unique()))
dropdown_B = pn.widgets.Select(name='B', options=list(df['B'].unique()))

@pn.depends(dropdown_A.param.value, dropdown_B.param.value)
def filter_dataframe(value_A, value_B):
    # Filtering the dataframe based on dropdown selection
    temp_df = df[(df['A'] == value_A) & (df['B'] == value_B)]
    
    # Creating a plotly figure
    fig = px.bar(temp_df, x=temp_df.index, y='C', title=f'Data for A={value_A} and B={value_B}')
    return pn.pane.Plotly(fig)

widgets = pn.Column("<b>Filters</b>", dropdown_A, dropdown_B)
app = pn.Row(widgets, filter_dataframe)
app

**Comment by rekcahpassyla** [comment]
Further version information about the above- 

plotly version 5.4.1
panel version 0.14.4

Last known good version of jupyterlab that worked with the above is 3.6.3

**Comment by fcollonval** [comment]
Closing as fixed in panel 1.1.1 according to the [release notes](https://panel.holoviz.org/about/releases.html#version-1-1-1).
